src/Widgets/AddressWidget/AddressWidgetComponent.tsx

```diff
@@ -40,21 +40,11 @@ const Logo = connect(() => {
 
 const Address = connect(
   ({ addressWidget }: { addressWidget: AddressWidgetInstance }) => {
-    let localityRegionPostalCode: string[]
-
-    if (addressWidget.get('addressFormat') === 'GER') {
-      localityRegionPostalCode = [
-        addressWidget.get('locationPostalCode'),
-        addressWidget.get('locationLocality'),
-        addressWidget.get('locationRegion'),
-      ]
-    } else {
-      localityRegionPostalCode = [
-        addressWidget.get('locationLocality'),
-        addressWidget.get('locationRegion'),
-        addressWidget.get('locationPostalCode'),
-      ]
-    }
+    const localityRegionPostalCode = [
+      addressWidget.get('locationPostalCode'),
+      addressWidget.get('locationLocality'),
+      addressWidget.get('locationRegion'),
+    ]
```

Comment:
* [ ] FYI the variable name is now out of order (component-wise)
* [ ] FYI now we only have one address line we could think about turning this into a single string attribute with any space-separated content. Not sure how much we/editors gain from making this structured.